cloudsync_onedrive.py

```diff
@@ -832,13 +838,14 @@ def _info_from_rest(self, item, root=None):
 
         pid = item["parentReference"].get("id")
         name = item["name"]
+        size = item["size"]
```

Comment:
have you tested how the size field operates in the context of a folder?

Reply:
I did a quick manual test and it works. I think a test should go in the cloudsync/test/test_provider.py to ensure the size field works across providers. We had talked about wrapping this in with SmartSync.